### PR TITLE
docs(readme): add latest release badge

### PR DESCRIPTION
**Pull request by R3DRUN3**
- [X] Read the [contribution guidelines](https://unikraft.org/docs/contributing/) regarding submitting new changes to the project;
  - [X] Updated relevant documentation.

### Description of changes
*SAFE TO MERGE PR*
Added latest release badge to readme:
GitHub release (latest SemVer)

### CODE REVIEW

File: README.md
Comment (nderjung) [inline README.md]:
```diff
- ![GitHub release (latest SemVer)](https://img.shields.io/github/v/release/unikraft/kraftkit)
+ ![Latest release](https://img.shields.io/github/v/release/unikraft/kraftkit)
```

**Review by nderjung — CHANGES_REQUESTED** [review]
Hi @R3DRUN3! Thanks for the contribution! This is a great idea. Just one inline suggestion and need you to force-push with a sign-off (`git commit -s`) before we can merge.

**Comment by R3DRUN3** [comment]
> Hi @R3DRUN3! Thanks for the contribution! This is a great idea. Just one inline suggestion and need you to force-push with a sign-off (`git commit -s`) before we can merge.

Hi @nderjung, done! 👍

File: README.md
Comment (R3DRUN3) [reply README.md]: Hi @nderjung, done! 👍

**Comment by R3DRUN3** [comment]
> Hi @R3DRUN3! Thanks for the contribution! This is a great idea. Just one inline suggestion and need you to force-push with a sign-off (`git commit -s`) before we can merge.

Hi @nderjung, done! 👍

**Review by nderjung — CHANGES_REQUESTED** [review]
Hi @R3DRUN3, please squash both commits together; simply 1 commit together and then force push. The workflow should be:

```
git stash;
git rebase -i HEAD~2;
```
Now, in the editor:
```diff
- pick a7aaac7 docs(readme): add latest release badge
- pick 1c0fd6f docs(readme): @nderjung inline suggestion
+ r a7aaac7 docs(readme): add latest release badge
+ f 1c0fd6f docs(readme): @nderjung inline suggestion
```

Then edit the message to be:
```
doc(readme): add latest release badge

Signed-off-by: ...
```

Add your name and email at `...` in the format `Name <email@com>`

Once the editor closes, force push to the branch:
```
git stash pop;
git push -u --force origin staging
```

**Comment by R3DRUN3** [comment]
@nderjung  ok, now it should be good 😊

**Comment by nderjung** [comment]
Hi, just one thing, it's `doc` instead of `docs`! :)

**Pull request merged**
